mllib/src/test/scala/org/apache/spark/ml/evaluation/MulticlassClassificationEvaluatorSuite.scala

```diff
@@ -40,4 +41,51 @@ class MulticlassClassificationEvaluatorSuite
   test("should support all NumericType labels and not support other types") {
     MLTestingUtils.checkNumericTypes(new MulticlassClassificationEvaluator, spark)
   }
+
+  test("Multiclass Classification Evaluator") {
+    val labelAndScores = spark.createDataFrame(Seq(
```

Comment:
True - but it is not done for `BinaryClassificationEvaluatorSuite` - I guess we could add tests for binary too. At some point it is likely that the impl will be moved and amended in `ml`. But the tests could also be added if/when that happens.

So this is a neutral change, not really necessary right now. 

Perhaps let's revisit after 2.0?